github-issues.go

```diff
@@ -129,10 +125,6 @@ func populateIssues(url string) {
 		}
 		if flag != 1 {
 			gIssues = append(gIssues, eachGitIssue)
-			flag = 0
-		} else {
-			// this is not an open issue so don't do anything but continue.
-			flag = 0
```

Comment:
It needs to be inside the for loop. Otherwise github PRs will get pulled into the issue table.